homeassistant/components/emoncms/coordinator.py

```diff
@@ -19,13 +18,12 @@ def __init__(
         self,
         hass: HomeAssistant,
         emoncms_client: EmoncmsClient,
-        scan_interval: timedelta,
     ) -> None:
         """Initialize the emoncms data coordinator."""
         super().__init__(
             hass,
-            _LOGGER,
+            LOGGER,
             name="emoncms_coordinator",
             update_method=emoncms_client.async_list_feeds,
```

Comment:
Can we move the updating method to `_async_update_data` and properly catch exceptions and raise `UpdateFailed` when that happens? This way we can omit the connection check in `__init__.py` because `coordinator.async_config_entry_first_refresh()` handles this then

Reply:
done